api/api-doc/storage_service.json

```diff
@@ -511,6 +511,21 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/cdc_streams_check_and_repair",
```

Comment:
nitpick (but you can ignore): maybe the word "repair" is a problem because it already means something else in Scylla?